assignments/callbacks.js

```diff
@@ -2,29 +2,75 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+  cb(arr);
 }
 
+function first(items) {
+  console.log(items[0]);
+}
+firstItem(items, first);
+
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+
+  cb(arr);
+}
+
+function length(items) {
+  console.log(items.length);
 }
+getLength(items,length);
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+
+  cb(arr);
+}
+
+function lastitem(items,last) {
+  console.log(items[3]);
```

Comment:
a more dynamic way to get the last item in the `items` array would be `items[items.length - 1]`